ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/snmp-settings.j2

```diff
@@ -1,6 +1,21 @@
 {# eos - SNMP Settings#}
 {% if snmp_server is arista.avd.defined %}
 !
+{%     if snmp_server.engine_ids is arista.avd.defined %}
+{%         if snmp_server.engine_ids.local is arista.avd.defined %}
+snmp-server engineID local {{ snmp_server.engine_ids.local }}
+{%         endif %}
+{%         for engine_id in snmp_server.engine_ids.remotes %}
```

Comment:
When you move this for loop after line143, please make sure you add defined test for ```snmp_server.engine_ids.remotes``` instead of ```snmp_server.engine_ids```

Reply:
there is no need to test for existence - for loop on an empty list does not print anything

Reply:
Should I test existence for both?

Reply:
Nope - interesting ok :)! will do